docs/architecture/decisions/0003-where-to-publish-container-image.md

```diff
@@ -0,0 +1,34 @@
+# 1. Where to publish the container image
+
+Date: 2022-09-07
+
+## Status
+
+Proposed
+
+## Context
+
+This project is packaged as a container image. Choose the container registry where deliver the image.
```

Comment:
```suggestion
This project is packaged as a container image. Choose the container registry where deliver the Docker image.
```

Reply:
I didn't use `Docker` to be technology agnostic. What do you think to use the term **container** instead of Docker?

Reply:
Yep, container image is ok for me! 🍻